--- a/conf/cuckoo.conf
+++ b/conf/cuckoo.conf
@@ -17,7 +17,7 @@
 # Specify the name of the machinery module to use, this module will
 # define the interaction between Cuckoo and your virtualization software
 # of choice.
-machinery = virtualbox
+machinery = kvm
 
 # Enable creation of memory dump of the analysis machine before shutting
 # down. Even if turned off, this functionality can also be enabled at
